server/src-lib/Hasura/Server/Auth.hs

```diff
@@ -1,4 +1,6 @@
-{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
`RecordWildCards` and `GeneralizedNewtypeDeriving` is already part of cabal's `default-extensions`.

Reply:
Should I remove all references to these extensions? Or just from my PR?